### ISSUE TITLE
Installing @web3-storage/w3up-client via yarn failed

### ISSUE THREAD

**Issue by zhanleewo** [web3-storage/w3up#1154]
Error 
```
git ls-remote --tags --heads git://github.com/web3-storage/one-webcrypto.git
fatal: unable to connect to github.com:
github.com[0: 140.82.112.4]: errno=Connection timed out
```

Could you change `git://github.com/web3-storage/one-webcrypto` to `https://github.com/web3-storage/one-webcrypto`
in `web3-storage/w3up/tree/main/packages/access-client/package.json`
```[tasklist]
### Tasks
- [ ] https://github.com/web3-storage/w3up/pull/1157
```

**Comment by gobengo** [comment]
@zhanleewo I released `@web3-storage/access` 18.0.2 with the change you requested, and I think it helped. I was able to do
```
bengo@bengo ~/yarn-test-1700259829 ⚡  npx yarn add '@web3-storage/w3up-client'
```

**Comment by zhanleewo** [comment]
thanks a lot
